.github/docker/centreon-web/alma8/Dockerfile

```diff
@@ -21,6 +21,20 @@ dnf config-manager --set-disabled 'centreon-plugin*unstable*' --set-disabled 'ce
 
 dnf install -y /tmp/packages-centreon/centreon-*.rpm
 
+###################
+# install plugins #
+###################
+
+dnf install -y centreon-plugin-Operatingsystems-Linux-Snmp nrpe centreon-plugin-Operatingsystems-Linux-Local.noarch nagios-plugins-nrpe
```

Comment:
considering the amount of dockerize (using Dockerfile) being run in a single day compared to Dockerfile.dependencies, I think it would be best to outsource the installation of plugins to https://github.com/centreon/centreon/blob/develop/.github/docker/centreon-web/bookworm/Dockerfile.dependencies
to avoid generating undesirable traffic with packages repositories

The configuration could remain here though